py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -0,0 +1,486 @@
+"""This module defines the class Dr16ExpectedFlux"""
+import logging
+import multiprocessing
+
+import fitsio
+from astropy.io import fits
+import numpy as np
+from scipy.interpolate import interp1d
+import healpy
+from pkg_resources import resource_filename
+
+from picca.delta_extraction.astronomical_objects.forest import Forest
+from picca.delta_extraction.astronomical_objects.pk1d_forest import Pk1dForest
+from picca.delta_extraction.errors import ExpectedFluxError
+from picca.delta_extraction.expected_flux import ExpectedFlux
+
+accepted_options = ["input directory", "iter out prefix",
+                    "num processors",
+                    "var lss binning"]
+
+defaults = {
+    "iter out prefix": "delta_attributes",
+}
+
+
+class TrueContinuum(ExpectedFlux):
+    """Class to the expected flux calculated used the true continuum unabsorbed contiuum
+    for mocks. It uses var_lss pre-computed from mocks and the mean flux estimated from 
```

Comment:
This doc string seems broken, both in the beginning and ending of sentences.

Reply:
> By the way @andreicuceu , @alxogm , I noticed that the current code hard-codes the value of (z)
> 
> ```
>             mean_optical_depth = np.ones(forest.log_lambda.size)
>             tau, gamma, lambda_rest_frame = 0.0023, 3.64, 1215.67
> ```
> 
> We should be reading this from the raw skewers instead.

I've seen you open an issue so I'll mark here as resolved, and fix after the first merge.